src/cli/README.md

```diff
@@ -680,3 +681,124 @@ Remove an IEEE 802.15.4 Extended Address from the whitelist.
 > whitelist remove dead00beef00cafe
 Done
 ```
+
+### diag
```

Comment:
Would it make sense to put the primary documentation in `src/diag/README.md` and just have a relative link here?